model/player.go

```diff
@@ -21,8 +25,10 @@ type Player struct {
 type Players []Player
 
 type PlayerRepository interface {
+	rest.Persistable
+	ResourceRepository
```

Comment:
Any particular reason to add these?

Reply:
Yes; to expose the APIs for testing (and as far as I can tell, the player implements all of these methods already)

Reply:
Ah, I see. What I usually do in these cases is something like this:
```golang
var repo *playerRepository
repo = NewPlayerRepository(ctx, database).(*playerRepository)
```
This avoids changing the "production" code just to satisfy the tests. Sometimes it is inevitable, but I strive to keep the prod code clean